### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by larsmm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,12 +48,17 @@ cp firmware/configs/sha2017_defconfig firmware/sdkconfig
 ```
 
 # Build instructions
+Set the path to esp32-toolchain (you have to repeat that on every new terminal):
+```
+export PATH="$PATH:/.../ESP32-platform-firmware/xtensa-esp32-elf/bin"
```
Comment (renzenicolai) [inline README.md]: What does the ... do? If it's a "fake" path then please replace it with something making that obvious :-)

For example:
export PATH="$PATH:/path/to/my/toolchain/xtensa-esp32-elf/bin"

**Review by renzenicolai — CHANGES_REQUESTED** [review]

**Comment by larsmm** [comment]
OK, please check again.

File: README.md [hunk 2]
```diff
@@ -48,12 +48,17 @@ cp firmware/configs/sha2017_defconfig firmware/sdkconfig
 ```
 
 # Build instructions
+Set the path to esp32-toolchain (you have to repeat that on every new terminal) (usually it is located in your ESP32-platform-firmware directory):
+```
+export PATH="$PATH:/path/to/my/toolchain/xtensa-esp32-elf/bin"
```
Comment (raboof) [inline README.md:53]: This used to be automatic: if you installed the toolchain into the root of the repo as documented above (`unzip -p toolchain/xtensa-esp32-elf-linux64.zip xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar | tar xvf -`) then it was picked up automatically by `set_env.sh`, but that was lost in #121 

While documenting this is an improvement, perhaps even better would be to move adding the toolchain to the end of the `PATH` into `build.sh`?

Comment (raboof) [reply README.md:53]: (I'll test+PR this tomorrow night unless anyone else beats me to it)

Comment (larsmm) [reply README.md:53]: Renze said this should be done by hand. I get this error from starting config.sh:

`./config.sh: line 2: set_env.sh: No such file or directory`

Comment (raboof) [reply README.md:53]: > Renze said this should be done by hand

Yes, this currently should be done by hand, but I propose to change the scripts so it no longer needs to be done by hand :smile: .

> I get this error from starting config.sh:
> `./config.sh: line 2: set_env.sh: No such file or directory`

That looks like a separate bug ;) - you can probably just remove the `source set_env.sh` line from `config.sh`

Comment (raboof) [reply README.md:53]: #129 should do it, could you give that a go?

**Pull request merged**